### PR TITLE
Feature was completed and it's ready to continue

### PR DESCRIPTION
**Pull request by cochabambinoski**
We implement the project required that consists of creating a simple web page that a user can add and remove books.
- Added descriptive readme.
- Functions and programming implemented to make the functionality.
- Implemented eslint.

### CODE REVIEW

**Comment by davitomix** [comment]
Hi @maelfosso, Thanks for the review.
We already added all the requirements you listed before.

- [ ]  Fixed error related to the stylesheet.

- [ ]  Fixed issue that makes the browser crash.

- [ ] Added buttons to delete and save the book items.

File: index.html [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+    integrity="sha384-Vkoo8x4CGsO3+Hhxv8T/Q5PaXtkKtu6ug5TOeNV6gBiFeWPGFN9MuhOf23Q9Ifjh"
+    crossorigin="anonymous"
+  />
+</head>
+<body>
+  <main id="mainContainer">
+      <div id="form">
+          <div id="titleForm">
+              <label for="titleInput">Title: </label>
+              <input type="text" id="titleInput" name="titleInput">
```
Comment (bellom) [inline index.html:20]: **Kindly do this before merging.....**

_**User shouldn't be able to create empty todo right?**_

You can apply `required` to all the input here
screenshot from your app below
image

Comment (davitomix) [reply index.html]: Done 😄

**Comment by bellom** [comment]
Good! 👍

**Pull request merged**